mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1063,9 +1063,9 @@ def Rock_BlockwiseGemmOp:
 // accounts for how we call these operations.
 defvar MfmaArgTypes = [F32,
                     VectorOfLengthAndType<[2, 4], [F32]>,
-                    VectorOfLengthAndType<[2, 4, 8], [F16, BF16]>,
-                    VectorOfLengthAndType<[4, 8], [I8, F8E5M2FNUZ, F8E4M3FNUZ]>];
-defvar MfmaResTypes = [VectorOfLengthAndType<[4, 16, 32], [F32, I32]>];
+                    VectorOfLengthAndType<[2, 4, 8, 16], [F16, BF16]>,
+                    VectorOfLengthAndType<[4, 8, 16], [I8, F8E5M2FNUZ, F8E4M3FNUZ]>];
+defvar MfmaResTypes = [VectorOfLengthAndType<[4, 8, 16, 32], [F32, I32, F16, BF16]>];
```

Comment:
So these are used in blockwisegemmV2 and threadwisegemmV2 which don't even have a verifier. My assumption was that it is ok to be not too strict here, because it's an internal operator and there is no risk in doing anything wrong because any length/type mismatch would be caught later.